Lib/test/test_posix.py

```diff
@@ -1025,8 +1025,10 @@ def _create_and_do_getcwd(dirname, current_path_length = 0):
     def test_getgrouplist(self):
         user = pwd.getpwuid(os.getuid())[0]
         group = pwd.getpwuid(os.getuid())[3]
-        self.assertIn(group, posix.getgrouplist(user, group))
-
+        try:
+            self.assertIn(group, posix.getgrouplist(user, group))
+        except OSError:
+            raise unittest.SkipTest("os.getgrouplist() is not available")
```

Comment:
I'm not sure that it's ok to ignore such error.

The bare minimum is to add a comment explaining the skip with a reference to the bpo.